master/docs/tutorial/firstrun.rst

```diff
@@ -110,16 +113,16 @@ Now that we are ready, we need to install buildbot:
   pip install --upgrade pip
   pip install 'buildbot[bundle]'
 
-Now that buildbot is installed, it's time to create the master:
+Now that buildbot is installed, it's time to create the master.
+``my_master`` represents a path to a directory, where future master will be created:
 
 .. code-block:: bash
 
-  buildbot create-master master
+  buildbot create-master my_master
```

Comment:
I think the rest of the tutorial is no longer consistent with the above command.

Reply:
Fixed.